### PR TITLE
day1+2

### PR DESCRIPTION
**Pull request by ShayRasooly**

### CODE REVIEW

File: Week1/Day1/Daily Challenge Build Up A String/code.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    print(user_string[:ind])
+    ind += 1
+
+
+import random as rd
+
+chars = list(user_string)
+rd.shuffle(chars)
+user_string_new = ''.join(chars)
+print(user_string_new)
```
Comment (ShayRasooly) [inline Week1/Day1/Daily Challenge Build Up A String/code.py:30]: Great!

File: Week1/Day1/Exercises XP Gold/code.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        if user_month in Spring:
+            print('The month is Spring')
+        elif user_month in Winter:
+            print('The month is Winter')
+        elif user_month in  Summer:
+            print('The month is Summer')
+        else:
+            print('The month is Autumn')
+        break
+
```
Comment (ShayRasooly) [inline Week1/Day1/Exercises XP Gold/code.py:40]: Great!

File: Week1/Day1/Exercises XP Ninja/code.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+#You can call python3 from any directory because it's typically installed as a
+# system-wide executable. This means that the operating system knows where to find the
+# python3 executable regardless of your current directory.
+
+
```
Comment (ShayRasooly) [inline Week1/Day1/Exercises XP Ninja/code.py:5]: what about PATH and alias?

File: Week1/Day1/Exercises XP Ninja/code.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+#print("b:", b) -- 10
+
+#Exercise 4 : How Many Characters In A Sentence ?
+my_text = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+print(len(my_text)) # including spaces
+
+#Exercise 5: Longest Word Without A Specific Character
+char = 'a'
+sp = [0]
+while char == char:
```
Comment (ShayRasooly) [inline Week1/Day1/Exercises XP Ninja/code.py:35]: this is the most elegant way you've found to create an infinite loop?

File: Week1/Day1/ExercisesXP/code.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    print('Even number')
+else:
+    print('Odd number')
+#Exercise 8 : What’s Your Name ?
+user_name = input('What’s Your Name?: ').lower()
+if user_name == 'mikhail' or user_name == 'michael':
+    print('You do not say! It is impossible!!!')
+#Exercise 9 : Tall Enough To Ride A Roller Coaster
+user_height = int(input('What is your height(in inches): '))
+user_height_cm = user_height*2.54
```
Comment (ShayRasooly) [inline Week1/Day1/ExercisesXP/code.py:42]: nice conversion!

File: Week1/Day1/Exercises XP Ninja/code.py (hunk 4 above)
Comment (TiurinMikhail) [reply Week1/Day1/Exercises XP Ninja/code.py:35]: We can also use :
while char:
...

or if we do not want to use char at all

while True:
...

File: Week1/Day1/Exercises XP Ninja/code.py (hunk 3 above)
Comment (TiurinMikhail) [reply Week1/Day1/Exercises XP Ninja/code.py:5]: The PATH environment variable is essential for running commands and scripts from any location in the Command Prompt or PowerShell without specifying the full path to the executable file
In CMD or shell environments, aliases are shortcuts or alternate names for commands. Users can define custom aliases to abbreviate commonly used commands or to create shortcuts for longer commands.

**Pull request merged**